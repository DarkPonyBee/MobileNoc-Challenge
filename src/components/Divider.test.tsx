import React from 'react';
import {Divider} from './Divider';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

describe('Divider', () => {
  it('should render without issue', () => {
    const component = shallow(<Divider />);
    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});

Enzyme.configure({adapter: new Adapter()});
