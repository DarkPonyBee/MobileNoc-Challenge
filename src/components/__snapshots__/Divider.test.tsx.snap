// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider should render without issue 1`] = `
<View
  style={
    Object {
      "height": 5,
    }
  }
/>
`;
